### DISCUSSION TITLE
Thanks for doing this

### DISCUSSION THREAD

**Discussion by emmanuelthegrey** [Amka78/owl-of-athena#40] [General]
I was literally getting ready to do the exact same thing. I can see you stopped working on this a few years back. I am wondering if its still viable and if you have any plans of starting it back up. I would love to help were possible. I am about to try and connect it to my dreamband now 🤞

**Comment by Rajabox** [comment]
> I was literally getting ready to do the exact same thing. I can see you stopped working on this a few years back. I am wondering if its still viable and if you have any plans of starting it back up. I would love to help were possible. I am about to try and connect it to my dreamband now 🤞

Have you figured out how to run the program?

**Reply by rasimmonds76** [reply]
I am getting errors when trying to run this, is this still being developed?

**Reply by Rajabox** [reply]
I don't think so
